internal/providers/terraform/azure/event_hubs_namespace.go

```diff
@@ -0,0 +1,165 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMEventHubsNamespaceRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_eventhub_namespace",
+		RFunc: NewAzureRMEventHubs,
+		Notes: []string{"Premium namespaces are not supported by Terraform."},
+	}
+}
+
+func NewAzureRMEventHubs(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var events, unknown *decimal.Decimal
+	sku := "Basic"
+	meterName := "Throughput Unit"
+	region := lookupRegion(d, []string{})
+
+	if d.Get("sku").Type != gjson.Null {
+		sku = d.Get("sku").String()
+	}
+
+	capacity := decimal.NewFromInt(1)
```

Comment:
https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/internal/services/eventhub/eventhub_namespace_resource.go#L77-L80 shows the default value, not sure why the TF docs didn't say that.

Reply:
yes, the terraform doesn't have it.